packages/components/.storybook/preview.tsx

```diff
@@ -1,12 +1,60 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
+import React, { ComponentType } from 'react';
 
-import '@frontify/fondue-tokens/styles';
-import type { Preview } from '@storybook/react';
 import '../src/styles.scss';
+import "@frontify/fondue-tokens/themeProvider/styles"
+import type { Preview, Decorator, StoryContext } from '@storybook/react';
```

Comment:
Linter should have marked that as an issue 🤔 
```suggestion
import { type Preview, type Decorator, type StoryContext } from '@storybook/react';
```